### PR TITLE
Stories: change the create sheet order

### PR DESCRIPTION
**Pull request by leandroalonso**
This PR moves the "Story post" option to be the first one shown in the "Create New" sheet.

### To test

1. Tap the FAB under "My Site" and "Blog posts"
2. Make sure "Story post" is the first option

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
UI change, no automated tests were added.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by leandroalonso** [comment]
cc @renanferrari

**Review by bjtitus — APPROVED** [review]
✅ Verified that the Story item is first in the list.

Looks good to me 👍

**Comment by mzorz** [comment]
👋 @leandroalonso - a discussion held on the [mirrored PR for WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/pull/14557) brought up the issue of having a different order for the creation options list on the My Site and the Posts list screens.

**tl;dr** both lists should be consistent, hence if we change the order in which things appear on the My Site screen, we should also make the change on the Posts list side of things 🙏 

cc @renanferrari

**Comment by jkmassel** [comment]
👋 We're freezing 17.3 today, so this PR is being bumped to 17.4

**Comment by leandroalonso** [comment]
@mzorz Thanks for the context!

I'll update the PR.

**Comment by leandroalonso** [comment]
@bjtitus can I bother you again, sir?

**Comment by bjtitus** [comment]
✅ Verified that the Story Post option appears first in the Blog Posts list.

Screen Shot 2021-05-06 at 10 41 24 AM

**Review by bjtitus — APPROVED** [review]
